DepositTransaction.java

```diff
@@ -1,17 +1,15 @@
 /**
  * DepositTransaction class handles deposit operations.
- * Increases the account balance by the deposit amount.
  */
 public class DepositTransaction extends BaseTransaction {
 
-    // Constructor
     public DepositTransaction(double amount, Calendar date, String transactionID) {
         super(amount, date, transactionID);
     }
 
     @Override
     public void apply(BankAccount ba) {
```

Comment:
Hi @Vankalondu,

I will merge this but please ensure you create another PR that implements this function to have the Logic in the Deposit Transaction class and not in the BankAccount class.

But your logic should be functional